src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.props

```diff
@@ -134,6 +134,12 @@ Copyright (c) .NET Foundation. All rights reserved.
     <ToolDepsJsonGeneratorProject>$(MSBuildThisFileDirectory)GenerateDeps\GenerateDeps.proj</ToolDepsJsonGeneratorProject>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(NETCoreSdkBundledMSBuildInformationProps)' == ''">
```

Comment:
I'd suggest putting this next to the import of Microsoft.NETCoreSdk.BundledVersions.props, which happens to be in Microsoft.NET.Sdk.DefaultItems.props.  Any reason not to do that?

Reply:
My instinct was 'import as soon a possible', but I think it would be fine to move this beside DefaultItems